cobra/io/mat.py

```diff
@@ -25,18 +25,18 @@
 
 
 # precompiled regular expressions
-_bracket_re = re.compile(r"\[[a-z]\]$")
-_underscore_re = re.compile(r"_[a-z]$")
+_bracket_re = re.compile(r"\[[a-z]{1,3}\]$")
```

Comment:
Is there any reason not to match everything inside of the square brackets?

```suggestion
_bracket_re = re.compile(r"\[[a-z]+\]$")
```

Reply:
No there is not I form the yeast models perspective. 

For the underscore notation I just wanted to play it save so met names without compartment names in the id are not accidentally read as such.